corehq/form_processor/submission_post.py

```diff
@@ -160,6 +164,7 @@ def run(self):
             else:
                 case_db_cache = self.interface.casedb_cache(domain=domain, lock=True, deleted_ok=True, xforms=xforms)
 
+            known_submission_error = False
```

Comment:
seems like this should be `None` and not false since later you're setting it to a String